diff --git a/arch/arm64/boot/dts/qcom/qcs8550-retroidpocket-rpnova.dts b/arch/arm64/boot/dts/qcom/qcs8550-retroidpocket-rpnova.dts
index b21c84b..0345a26 100644
--- a/arch/arm64/boot/dts/qcom/qcs8550-retroidpocket-rpnova.dts
+++ b/arch/arm64/boot/dts/qcom/qcs8550-retroidpocket-rpnova.dts
@@ -12,9 +12,14 @@
 	compatible = "retroidpocket,rpnova", "qcom,qcs8550", "qcom,sm8550";
 };
 
+&backlight {
+	status = "disabled";
+};
+
 &mdss_dsi0 {
 	panel@0 {
 		compatible = "il97680a,rpnova";
+		/delete-property/ backlight;
 		/delete-property/ rotation;
 	};
 };
